include/exadg/solvers_and_preconditioners/preconditioners/preconditioner_amg.h

```diff
@@ -53,8 +53,7 @@ class PreconditionerTrilinosAMG : public PreconditionerBase<Number>
 #endif
 
 public:
-  PreconditionerTrilinosAMG(Operator const & op, AMGData data = AMGData())
-    : pde_operator(op), amg_data(data)
+  PreconditionerML(Operator const & op, AMGData data = AMGData()) : pde_operator(op), amg_data(data)
```

Comment:
I think here we should only pass the ML related data for a better separation of concerns.

Reply:
Done and also the same for `BoomerAMG`.